src/main/java/halberstam/wordle/WordleDictionaryTest.java

```diff
@@ -0,0 +1,30 @@
+package halberstam.wordle;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+class WordleDictionaryTest {
+
+    @Test
+    void getDefinition() throws IOException {
+        //given
+        WordleDictionary wordle = new WordleDictionary("C:\\Users\\Elky\\Documents\\Touro\\Spring 2023\\Swimmer 1\\dictionary.txt");
```

Comment:
put dictionary.txt into your project and do not use an absolute path, use a relative path.